charts/psmdb-db/production-values.yaml

```diff
@@ -0,0 +1,130 @@
+# Default values for pxc-cluster.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Platform type: kubernetes, openshift
+# platform: kubernetes
+
+# Cluster DNS Suffix
+# DNSsuffix: .svc.cluster.local
+
+nameOverride: ""
+fullnameOverride: ""
+
+pause: false
+allowUnsafeConfigurations: false
+# updateStrategy: SmartUpdate
+
+image:
+  repository: percona/percona-server-mongodb-operator
```

Comment:
repository should be: percona/percona-server-mongodb
tag: 4.2.8-8

Reply:
done